app/uk/gov/hmrc/plasticpackagingtax/registration/views/amendment/group/list_group_members_page.scala.html

```diff
@@ -61,16 +58,21 @@
                 ))
             ).getOrElse(Seq())
 }
-
+@*
+*<!--todo title should change for an errored form?-->
```

Comment:
Don't think so.

Reply:
Its usually a GDS that we do for accessibly, as for screen readers they would be expecting a new page, updating the title to say theres an error quickly gives reason to why they havent progressed on the journey. 
Probably a wider issue we will need to look in to for other pages too

Reply:
Fair. Probably one for Tom McArdle and Joel to take a look at.